README.md

```diff
@@ -193,6 +193,76 @@ public class SomeOtherTest {
     }
 ```
 
+##### Mapping ports
+
+Often there is a need to connect to a container via a certain port. To achieve this goal, you need to know the mapped 
+port of the port that was specified from the Java code. To get the mapped port use the `getMappedPort(...)` method
+of testcontainers API. 
+
+As an example, consider the following Java code:
+
+```java
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.ExternalResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.TarantoolCartridgeContainer;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class AppTest {
+
+    private final Logger logger = LoggerFactory.getLogger(AppTest.class);
+
+    private static final GenericContainer<?> container =
+            new TarantoolCartridgeContainer("cartridge/instances.yml", "cartridge/topology.lua")
+                    .withDirectoryBinding("cartridge")
+                    .withRouterHost("localhost")
+                    .withRouterPort(3301)
```

Comment:
It's better to add an example about other ports. Not only about the main 3301. What if I want to connect to another router

Reply:
Fixed